web-marketplace/src/components/organisms/BuyFiatModal/BuyFiatModal.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box } from '@mui/material';
+
+import ContainedButton from 'web-components/src/components/buttons/ContainedButton';
+import OutlinedButton from 'web-components/src/components/buttons/OutlinedButton';
+import Card from 'web-components/src/components/cards/Card';
+import SellOrderNotFoundIcon from 'web-components/src/components/icons/SellOrderNotFoundIcon';
+import Modal from 'web-components/src/components/modal';
+import { Title } from 'web-components/src/components/typography';
+
+import { UseStateSetter } from 'types/react/use-state';
+
+import { BuyFiatModalContent, FiatModalStateType } from './BuyFiatModal.types';
+
+interface BuyFiatModalProps extends BuyFiatModalContent {
+  fiatModalState: FiatModalStateType;
+  onClose: UseStateSetter<FiatModalStateType>;
+  handleClick: (action: string | null) => void;
+}
+
+export const BuyFiatModal = ({
```

Comment:
it's a bit weird to call it `BuyFiatModal` when it's not only about fiat credits
maybe something like `BuyWarningModal`?